Formula/joern.rb

```diff
@@ -35,8 +35,8 @@ def install
   end
 
   test do
-    ENV["JAVA_HOME"] = Formula["openjdk"].opt_prefix
-    ENV.append_path "PATH", Formula["openjdk"].opt_bin
+    ENV["JAVA_HOME"] = Formula["openjdk@17"].opt_prefix
+    ENV.append_path "PATH", Formula["openjdk@17"].opt_bin
```

Comment:
Kinda weird that we have to set these at all. It seems to me that we actually want to use `write_env_script` rather than `write_exec_script` at `install`?